ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/platform.cfg

```diff
@@ -33,6 +33,7 @@ platform sand lag hardware-only
 platform sand lag mode 512x32
 platform sand forwarding mode arad
 platform sand multicast replication default ingress
+platform sfe data-plane cpu allocation maximum 42
```

Comment:
This command was not working on several HW platforms I tested.

Reply:
Will likely only work on AWE and CloudEOS

Reply:
@ClausHolbechArista - should we then mention in the description that this command support is platform dependent maybe?

Reply:
Yeah, maybe just add it on the platform key, since everything below that is highly platform specific.